.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: UI Tests
+
+on:
+  workflow_dispatch:
+  deployment_status:
+
+env:
+  GIT_PAT: ${{ secrets.GIT_PAT }}
+  MNEMONIC: ${{ secrets.MNEMONIC }}
+  MNEMONIC_MAINNET: ${{ secrets.MNEMONIC_MAINNET }}
+  PATH_COSMOSTATION: './wallets/cosmostation'
+  PATH_KEPLR: './wallets/keplr'
+  PATH_LEAP: './wallets/leap'
+  PATH_METAMASK: './wallets/metamask' 
+  WALLET_PASSWORD: ${{ secrets.WALLET_PASSWORD }}
+
+jobs:
+  github-actions-testnet:
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: 'main' 
```

Comment:
Won't this run tests on the `main` branch code, not the PR code?

When I've wanted to target the PR branch i've used the current head SHA
```
          ref: ${{ github.event.pull_request.head.sha }}
```
https://github.com/duality-labs/duality-web-app/blob/v2.2.1/.github/workflows/deploy_preview.yml#L19-L20

Reply:
ok, yes it makes sense that the build created from the branch is tested. but the end-2-end tests main branch is downloaded in the _next_ step, not this one. Maybe the checkout step isn't needed? 

To clone just the main branch we can use: `git clone <url> --branch main --single-branch`